src/internal.rs

```diff
@@ -272,6 +272,43 @@ pub(crate) struct TaskThreadData {
     pub delayed_fg: RwLock<TaskThreadDataDelayedFg>,
 }
 
+impl TaskThreadData {
+    /// When we finish a frame we advance `first`, and we must also adjust `cur`,
+    /// because `cur` is an offset of `first`.
+    /// We may also need to adjust `reset_task_cur`.
+    ///
+    /// The task thread lock must be held and there must be more than 1 frame context.
```

Comment:
Can you link to the task thread lock field directly?

Reply:
👍  will do